tests/model_registry/rest_api/test_model_registry_secure_db.py

```
@@ -0,0 +1,96 @@
+        Deploys Model Registry with a secure MySQL DB (SSL/TLS), registers a model, and checks functionality.
+        Uses a CA bundle file for SSL verification by passing it directly to the verify parameter.
+        """
+        model_name = generate_random_name(prefix="model-rest-api")
+        MODEL_REGISTER_DATA["register_model_data"]["name"] = model_name
+        result = register_model_rest_api(
+            model_registry_rest_url=model_registry_rest_url,
+            model_registry_rest_headers=model_registry_rest_headers,
+            data_dict=MODEL_REGISTER_DATA,
+            verify=local_ca_bundle,
+        )
+        assert result["register_model"].get("id"), "Model registration failed with secure DB connection."
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)